app/views/migration/_form.html.erb

```diff
@@ -86,6 +86,7 @@
     <% end %>
 
     <span class="required-fields-hint"><%= t('global.forms.hints.required-fields') %></span>
-    <%= story.button(:submit) %>
+    <span class="error global hidden"><%= t('global.forms.validation-errors.form-has-errors') %></span>
+    <%= story.button(:submit, class: 'blue') %>
```

Comment:
Do we really want to set class to "blue"? Isn't that a bit brittle? As in, should colours change, references to the classes will need to change too.

Reply:
It refers to $eu-blue, a standard colour.  All the names are quite short - I just added this one to the defaults (see link below) but I'll rename this one to eu-blue to be clearer

http://styleguide.europeana.eu/patterns/atoms-buttons-buttons/atoms-buttons-buttons.html